src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -634,21 +634,6 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
                     downQueries.push(new Query(`ALTER TABLE ${this.escapePath(table)} RENAME CONSTRAINT "${newPkName}" TO "${oldPkName}"`));
                 }
 
-                // rename column sequence
```

Comment:
Wouldn't dropping a lot of this code be bad because we'd never undo it and remove the sequence?

Like if someone upgrades to this version and have an existing database wouldn't they be stuck & the migration code would fail?

How do we migrate from the v9 compatible code to newer code?